pkg/config/api_server_config.go

```diff
@@ -65,6 +69,34 @@ func NewAPIServerConfig(ctx context.Context) (*APIServerConfig, error) {
 	return &config, nil
 }
 
+// AllowedTokenPublicKeys returns a map of 'kid' to the KMS KeyID reference.
+// This represents the keys that are allowed to be used to verify tokens,
+// the TokenSigningKey/TokenSigningKeyID.
+func (c *APIServerConfig) AllowedTokenPublicKeys() map[string]string {
+	{
+		c.mu.RLock()
+		if len(c.allowedTokenPublicKeys) != 0 {
```

Comment:
The RLock is not necessarily released. If, for example, c=nil, you panic, and don't release the lock. More robust is:

func() {
  c.mu.RLock()
  defer c.mu.RUnlock()
  if .....
}()